AUTHORS.rst

```diff
@@ -9,3 +9,4 @@ Contributors
 - Fabio Rosado (`@fabiorosado <https://github.com/fabiorosado>`_)
 - Niko Eckerskorn (`@kadrach <https://github.com/kadrach>`_)
 - Or Carmi (`@liiight <https://github.com/liiight>`_)
+- George Kontridze (`@gkze <https://github.com/gkze>`_)
```

Comment:
@gkze - I've added you as a contributor to `AUTHORS.rst`. Let me know if you'd prefer be removed from this file, or if want me to link a different account here (using your GitHub as default).